README.md

```diff
@@ -1,40 +1,146 @@
+This page will provide you with detailed instructions on how to set up all the necessary pre-requisites for the workshop.
+
+but first......
+
+We understand that some of these concepts and tools may be new to you. That's perfectly okay! We will guide you through each step and explain everything gradually. It's important to know that you don't have to grasp everything immediately, and you are not expected to become a professional software developer after this workshop.
+(However, it is important to mention, that we held a similar workshop last year, and as a result of the workshop, some participants were able to start pursuing a career in tech! so.. anything is possible really! 🚀)
+
+This day is mostly meant to give you a glimpse at what it means to build tech products, and to show you it is not as scary as it may seem.
+You don't need to understand every aspect in fine detail.
+Sometimes, following the instructions without fully understanding them and progressing to the next step is perfectly fine. In fact, even experienced developers often solve problems in this way!
+
+Please, don't be afraid to ask questions, thera are no bad questions. 
+
+Remember to **believe in yourself** and approach this day with a mindset of getting the most out of it.
+
+We strongly believe that we need more women in tech. Diversity (of any sort) brings more perspectives and ideas, and more perspectives and ideas mean better solutions, and better solutions means a better industry.
+
+If you're having problems, in any of the steps, please do not hesitate to contact your assigned mentor, she or he will be delighted to help!
+(btw, experienced developers, need to be independent as possible, and each of the following steps has plenty of information online, so you can also of course, try to Google the solution yourself,
+but only if you're feeling adventurous 🙂)
+
+Now let's get this show on the road! 
 
 ### Install PyCharm Community version
+
+**If you have a Windows Laptop:**
+https://www.jetbrains.com/pycharm/download/?section=windows
+
+**If you have an Apple laptop:**
 https://www.jetbrains.com/pycharm/download/#section=mac
 
+PyCharm is an IDE, what is an IDE you ask? good question.
+An IDE is a software program that helps people write and edit code more easily. It has tools to catch mistakes, make coding faster, and helps with organizing and testing your code. Essentially, it's a helpful tool for people learning or working with coding.
+
 ### Install Python
-https://www.python.org/downloads/
+
+What is Python you ask? good question.
+Python is a popular programming language known for its simplicity and versatility. It is used for web development, data analysis, AI, and more, and is beginner-friendly with a clean and readable syntax.
+(Of course Python is the writer's of this document favourite progmramming language, and I do expect it to be your favourite as well! Joking 😅)
```

Comment:
Very sympathetic language :) I suggest we use "we" instead of "I" throughout the course. So, it is coming from the ING family :)